### PR TITLE
Update dependencies, require node 16, remove swagger-ui-dist

### LINKED ISSUE #56: Update to node 16/es2022, remove swagger-ui

### PR DESCRIPTION
**Pull request by carlansley**
Fixes #56

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+    - "main"
```
Comment (le-cong) [inline .github/workflows/ci.yml:6]: indent

File: src/index.spec.ts [hunk 2]
```diff
@@ -24,12 +24,11 @@
  THE SOFTWARE.
  */
 
-import * as assert from 'assert';
+import * as assert from 'node:assert';
```
Comment (le-cong) [inline src/index.spec.ts:27]: import { strict as assert } from 'node:assert';

File: src/debug.ts [hunk 3]
```diff
@@ -45,30 +45,33 @@ export default function (module: string): (context: any, next: () => Promise<voi
...
 
     await next();
 
     // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-    const status = parseInt(context.status, 10);
-    const requestBody =
-      typeof context.request.body === 'undefined' ? context.request.body : JSON.stringify(context.request.body);
-    const responseBody = typeof context.body === 'undefined' ? context.body : JSON.stringify(context.body);
+    const status = Number.parseInt(context.status, 10);
+    const requestBody = body === undefined ? body : JSON.stringify(body);
```
Comment (le-cong) [inline src/debug.ts:58]: could be simplified as JSON.stringify(body)

File: package.json [hunk 4]
```diff
@@ -26,37 +26,31 @@
...
-    "@koa/cors": "^3.1.0",
-    "@koa/router": "^10.1.1",
+    "@koa/cors": "^3.4.1",
+    "@koa/router": "^12.0.0",
     "@types/koa-bodyparser": "^5.0.1",
-    "debug": "^4.3.3",
+    "debug": "^4.3.4",
     "koa": "^2.13.4",
     "koa-bodyparser": "^4.3.0",
     "koa-send": "^5.0.1",
```
Comment (le-cong) [inline package.json:35]: `koa-send` is no longer referenced in the code, should it be removed?

File: .github/workflows/coverage.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+name: Coverage
+
+on:
+  pull_request:
+    branches:
+    - "main"
```
Comment (le-cong) [inline .github/workflows/coverage.yml:6]: indent

File: .github/workflows/ci.yml (hunk 1 above)
Comment (carlansley) [reply .github/workflows/ci.yml:6]: not sure I understand this one, isn't this the same in all our ci.ymls?

Comment (le-cong) [reply .github/workflows/ci.yml:6]: it was copied like this in our ci.yml, but later on I received PR comments requesting adding the spaces for indention to make it looks like:

> on:
>   pull_request:
>     branches:
>       - "main"
>

Comment (le-cong) [reply .github/workflows/ci.yml:6]: I just checked again and it seems that the YAML spec doesn't enforce indention or consistency in terms of the usage of spaces, so I guess there is no impact whether we indent or not.
but for the purpose of keeping our code style consistent, I feel it won't hurt to add indention similar to our swagger yml

**Review by le-cong — CHANGES_REQUESTED** [review]
just a few minor comments

File: .github/workflows/coverage.yml (hunk 5 above)
Comment (carlansley) [reply .github/workflows/coverage.yml:6]: ok to fix these inconsistencies in a better way, have a PR for you https://github.com/checkdigit/prettier-config/pull/11

**Review by le-cong — CHANGES_REQUESTED** [review]
one minor comment

**Review by le-cong — APPROVED** [review]
lgtm

**Pull request merged**